components/tyk-pump/templates/deployment-pmp.yaml

```diff
@@ -157,6 +181,21 @@ spec:
                 key: {{ include "tyk-pump.mongo_url_secret_key" . }}
           - name: TYK_PMP_PUMPS_MONGO_META_MONGODRIVERTYPE
             value: "{{ default "mgo" .Values.global.mongo.driver}}"
+
+          # Mongo Graph Pump
+          - name: TYK_PMP_PUMPS_GRAPHMONGO_TYPE
+            value: "mongo-graph"
+          - name: TYK_PMP_PUMPS_GRAPHMONGO_META_COLLECTIONNAME
+            value: "tyk_graph_analytics_pump"
```

Comment:
@kofoworola We will need to make this configurable too.

Reply:
okay, will add that change